doc/userguide/rules/dns-keywords.rst

```diff
@@ -22,6 +22,17 @@ used in a rule all contents following it are affected by it.  Example:
 The dns_query keyword affects all following contents, until pkt_data
 is used or it reaches the end of the rule.
 
+dns_response
+------------
+
+The **dns_response** keyword allows you to match IPs in A record responses.
```

Comment:
only for A records? If I see it correct AAAA records are also supported?

Reply:
Yeah, I changed the code from the last PR. Its for all response types now and byte matching so no IP string either.

Reply:
Please fix.